splunk-otel-android/src/test/java/com/splunk/rum/ConnectionUtilTest.java

```diff
@@ -115,4 +121,40 @@ public void quiznos() {
         monitorCaptor.getValue().onLost(null);
         assertEquals(2, notified.get());
     }
+
+    @Test
+    @Config(sdk = Build.VERSION_CODES.LOLLIPOP)
```

Comment:
I discovered that you can give this an array of `sdk` entries to collapse to a single test.

Reply:
That won't work in this case unfortunately: https://github.com/signalfx/splunk-otel-android/pull/146#discussion_r704265207